text/0000-buildpacks-community.md

```diff
@@ -0,0 +1,156 @@
+
+
+# How it Works
+[how-it-works]: #how-it-works
+
+## Project Admission
+
+A project can be admitted to the Buildpacks community organization by creating a Github issue in the Buildpacks community repository. The issue should contain the following information:
+
+- Name of the project
+- Evidence for the above criteria
+- A list of maintainers for the project
+
+The above information will be structured into an appropriate issue template. The Buildpacks Leadership will review the issue and if the project meets the above criteria, the project will be added to the Buildpacks community organization. The Buildpacks Leadership will assign a steward team to the project and the team lead of the steward team will be responsible for ensuring that the project meets the above criteria.
+
+Once admitted, the team lead of the steward team will create a Github team for the project and add the project maintainers to the team and mark them as the team maintainers allowing them to add other maintainers. The existing team maintainers of the steward team will be added as maintainers to the project team.
+
+The team lead will also create a CODEOWNERS file for the project and add the project maintainers as the code owners.
+
+The project maintainers will be responsible for maintaining the list of project maintainers and ensuring that all project contributors are members of the Buildpacks community organization. 
+
+They will be able to add new Github members to the organization by creating a Github issue in the Buildpacks community [invites repository](https://github.com/buildpacks-community/invites).
+
+## Project Removal
+
+In case the project fails to meet the above criteria, the Buildpacks Leadership will work with the project maintainers to address the issues and if the project is still not ready, the project will be archived or removed from the Buildpacks community organization.
+
+## Project Graduation
+
+In case a project is deemed to be mature enough to be part of the core Buildpacks organization, the project maintainers can request for the project to be graduated to the core Buildpacks organization via the [Component Contribution RFC](https://github.com/buildpacks/rfcs/blob/main/text/0108-governance-component-maintainer-role.md). The Buildpacks TOC will review the request and if the project meets the criteria for graduation, the project will be moved to the core Buildpacks organization.
+
+
+# Migration
+[migration]: #migration
+
+N/A
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+
+N/A
+
+# Alternatives
+[alternatives]: #alternatives
+
+N/A
+
+# Prior Art
+[prior-art]: #prior-art
+
+- [CNCF Sandbox](https://www.cncf.io/sandbox-projects/)
+- [Paketo Community](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0008-paketo-community.md)
+- [Argoproj Labs](https://github.com/argoproj-labs)
+- [Crossplane Contrib](https://github.com/crossplane-contrib)
+
+# Unresolved Questions
```

Comment:
should we move the [pack-orb](https://github.com/buildpacks/pack-orb) here?

Reply:
I think so. I also think we might want to move the tekton plugin here as well?

Reply:
@AidanDelaney I might propose moving scafall here as well to allow you to be unblocked.